### PR TITLE
Enhanced project

### PR DESCRIPTION
**Pull request by ben-lab-zz**
All set. Please let me know if you have any questions!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,26 @@ Submit a pull request when you are finished and satisfied with your work.
 
 Use this area to communicate any thought processes, ideas, or challenges you encountered.
 
-*
-*
-*
+## Approach
+
+* I wanted to create a cross-device experience - make the site responsive.
+* I used the SMACSS method of naming selectors for modularity and scalability.
```
Comment (talentedmrjones) [inline README.md:42]: bonus!

File: README.md [hunk 2]
```diff
@@ -36,6 +36,26 @@ Submit a pull request when you are finished and satisfied with your work.
...
+
+## Challenges
+
+* The background pattern needed to be repeatable.
+* I had to create a @font-face for one of the fonts. If this were to be used for a client, I would check with legal before going into production about this.
+* For the Helvetica-Neue, I used Google Open Sans instead.
+* A personal challenge was learning Grunt throughout this exercise. I am very happy with the result!
```
Comment (talentedmrjones) [inline README.md:49]: Glad to hear it!

File: www/index.html [hunk 3]
```diff
@@ -3,39 +3,45 @@
...
+	</head>
+	<body>
+	  <!--[if lt IE 7]>
+	      <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
+	  <![endif]-->
+	  
+	  <div class="layout-container">
+	  	<div class="layout-rsvp"><a class="rsvp-btn" href="#" alt="Open Invitation">RSVP</a></div>
+	  	<div class="layout-polaroids">
+	  		<div class="polaroid title"><div class="polaroid-details title"><img src="img/min/event_title.png" alt="MDC Partners Polaroid Celebration" /></div><img src="img/min/photo_title.jpg" alt="MDC Partners - Polaroid Celebration" /></div>
```
Comment (talentedmrjones) [inline www/index.html:23]: Don't you think that the `alt` of this image, "MDC Partners Polaroid Celebration" should the top most `h1` on the page? It is after all what the page is all about...

File: www/index.html [hunk 4]
```diff
@@ -3,39 +3,45 @@
...
+	  		<div class="polaroid description"><p>Book Signing & Cocktail Hour of <i>Instant</i> — The Story of Polaroid by author Christopher Bonanos.</p><img src="img/min/photo_description.jpg" alt="Instant - The Story of Polaroid" /></div>
+	  		<div class="polaroid date"><p class="line1">From 6 to 8 or until the</p> <p class="line2">pictures get blurry</p><div class="polaroid-details date"><img src="img/min/event_details.png" alt="October 30th - The MDC Innovation Centre, 745 5th Ave., 19th Floor" /></div><img src="img/min/photo_date.jpg" /></div>
+	  		<div class="polaroid boot"><img src="img/min/photo_boot.jpg" alt="Photo of a Boot" /></div>
+	  		<div class="polaroid person"><img src="img/min/photo_portrait.jpg" alt="Portrait of a Person" /></div>
+	  	</div> <!-- end of polaroids -->
+	  	<div class="layout-invitation clearfix">
+	  		<div class="invitation-content">
+	  			<h1>Stop by for a minute.</h1>
```
Comment (talentedmrjones) [inline www/index.html:31]: Take a look at the outline of this document with this tool http://gsnedders.html5.org/outliner/

Does the outline make sense?

**Comment by talentedmrjones** [comment]
Just curious, why do you like SASS? Have you tried other pre-processors such as LESS or Stylus?

**Comment by talentedmrjones** [comment]
Are you familiar with "BEM" style CSS syntax?

**Comment by talentedmrjones** [comment]
How about Backbone.js or Require.js?

File: www/index.html (hunk 3 above)
Comment (ben-lab-zz) [inline www/index.html:23]: Good point!

**Comment by ben-lab-zz** [comment]
When I first learned pre-processor css, I went back and forth between SASS and LESS. I under up liking SASS more because of the syntax. Just a personal preference.

I've read articles etc. on "BEM" css syntax but never used it.

I've taken over backbone/require.js projects, but never from the ground up. It is next on my list of things to learn and implement in future projects.

**Comment by talentedmrjones** [comment]
Thanks for taking the time to complete the lab! I really appreciate all the effort you put into it. We will get back to you as soon as possible. Thanks again!
~Richard

**Pull request closed without merging**